jac-uk/admin
# #1984 QT Merit List. Download full merit list to excel

## What's included?
Enables QT Merit List data to be downloaded to excel

Closes #1984 
Closes #1973 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
On the QT Merit List screen press the Export button.
Check that each version of the download works and contains the correct data.
Check that the labels and instructions on screen make sense and align with accepted terminology.


## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
